Server/Infrastructures/DB/DbConnFactory.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Data;
+using Npgsql;
+
+namespace UserService.DB;
+
+public class DbConnFactory
```

Comment:
ada alasan kenapa kita mau pake dapper? btw di EF Core juga akan ada raw sql query untuk unampped type https://devblogs.microsoft.com/dotnet/announcing-ef8-preview-1/#raw-sql-queries-for-unmapped-types

Reply:
ga ada alesan khusus